core/domain/platform_feature_services_test.py

```diff
@@ -184,20 +233,51 @@ def test_evaluate_dev_feature_for_dev_server_returns_true(self) -> None:
             self.assertTrue(
                 feature_services.is_feature_enabled(self.dev_feature.name))
 
+    def test_evaluate_test_feature_for_dev_server_returns_true(self) -> None:
+        with self.swap(constants, 'DEV_MODE', True):
+            self.assertTrue(
+                feature_services.is_feature_enabled(self.test_feature.name))
+
     def test_evaluate_prod_feature_for_dev_server_returns_true(self) -> None:
         with self.swap(constants, 'DEV_MODE', True):
             self.assertTrue(
                 feature_services.is_feature_enabled(self.prod_feature.name))
 
+    def test_evaluate_dev_feature_for_test_server_returns_false(self) -> None:
+        with self.swap(constants, 'DEV_MODE', False):
+            with self.swap(feconf, 'OPPIA_PROJECT_ID', 'testserver'):
+                self.assertFalse(
+                    feature_services.is_feature_enabled(self.dev_feature.name))
+
+    def test_evaluate_test_feature_for_test_server_returns_true(self) -> None:
+        with self.swap(constants, 'DEV_MODE', False):
+            with self.swap(feconf, 'OPPIA_PROJECT_ID', 'testserver'):
+                self.assertTrue(
+                    feature_services.is_feature_enabled(self.test_feature.name))
+
+    def test_evaluate_prod_feature_for_test_server_returns_true(self) -> None:
+        with self.swap(constants, 'DEV_MODE', False):
+            with self.swap(feconf, 'OPPIA_PROJECT_ID', 'testserver'):
+                self.assertTrue(
+                    feature_services.is_feature_enabled(self.prod_feature.name))
+
     def test_evaluate_dev_feature_for_prod_server_returns_false(self) -> None:
         with self.swap(constants, 'DEV_MODE', False):
-            self.assertFalse(
-                feature_services.is_feature_enabled(self.dev_feature.name))
+            with self.swap(feconf, 'OPPIA_PROJECT_ID', 'oppiaserver'):
+                self.assertFalse(
+                    feature_services.is_feature_enabled(self.dev_feature.name))
+
+    def test_evaluate_prod_feature_for_prod_server_returns_false(self) -> None:
```

Comment:
Should be ``test_evaluate_test_feature_for_prod_server_returns_false``, right?

Reply:
Whoopsie, fixed, thanks!